chaincode-labs/chaincode-podcast/assumeutxo.md

```diff
@@ -300,11 +298,11 @@ This kind of change is interesting because when you think about it, from a threa
 Even people who are really experienced with the code base still have a really hard time determining, in some cases, whether a change is safe or not.
 So when you kind of crystallize your security assumptions in a place where almost everybody can review it...
 
-## Updating `assumevalid` value in the code
+### Updating `assumevalid` value in the code
 
 James O'Beirne: 00:18:51
 
-So for example, the way that we [update the `assumevalid` value](https://github.com/bitcoin/bitcoin/pull/9484), [(0.19](https://github.com/bitcoin/bitcoin/pull/17002), [0.18)](https://github.com/bitcoin/bitcoin/pull/15429), because we do that typically with every release, is somebody will post a modification of that value, and then a number of people will chime in on the pull request and say, yeah, so I used the node that I previously provisioned with the initial block download process, and I ran this RPC command, and it told me that this hash that you mentioned in the source code is actually in my chain.
+So for example, the way that we [update the `assumevalid` value](https://github.com/bitcoin/bitcoin/pull/9484), ([0.19](https://github.com/bitcoin/bitcoin/pull/17002), [0.18])(https://github.com/bitcoin/bitcoin/pull/15429), because we do that typically with every release, is somebody will post a modification of that value, and then a number of people will chime in on the pull request and say, yeah, so I used the node that I previously provisioned with the initial block download process, and I ran this RPC command, and it told me that this hash that you mentioned in the source code is actually in my chain.
```

Comment:
Hey @kouloumos, I think this change is incorrect, the original displays the rendered markdown as intended.